apps/docs/src/app/core/component-docs/info-label/examples/info-label-icon-numeric-example.component.html

```diff
@@ -1,4 +1,6 @@
-<span fd-info-label [type]="'numeric'" [color]="'2'" style="margin-right: 20px;">6</span>
-<span fd-info-label [type]="'numeric'" [color]="'3'" style="margin-right: 20px;">42k</span>
-<span fd-info-label [type]="'numeric'" [color]="'3'" style="margin-right: 20px;">14.7</span>
-<span fd-info-label [type]="'only-icon'" [color]="'5'" [glyph]="'hide'" style="margin-right: 20px;"></span>
+<div style="display: flex;">
```

Comment:
This is fixing the example, but not fixing the issue.
Once again this is a problem caused by `fd-reset` on icon (issue also exists in Fundamental Styles)